### PR TITLE
Minor Updates: text and env

### PR DESCRIPTION
**Pull request by DavidLiedle**
Updated the date, fixed some typos, and line-wrapped at 80 chars

### CODE REVIEW

**Comment by andrerom** [comment]
.env change should go to upstream (ezplatform)

File: app/Resources/views/pagelayout_footer.html.twig [hunk 1]
```diff
@@ -26,7 +26,7 @@
             <div class="row">
                 <div class="col-xs-offset-1 col-xs-5 space-top-half-more space-bottom-less">
                     <ul>
-                        <li class="white small-extra">Copyright &copy; 2015 eZ Systems AS (except where otherwise noted). All rights reserved. <a href="//ez.no/privacy-policy">Privacy Policy.</a> Powered by the <a href="//ez.no/ezpublish"> eZ Platform</a></li>
+                        <li class="white small-extra">Copyright &copy; 2016 eZ Systems AS (except where otherwise noted). All rights reserved. <a href="//ez.no/privacy-policy">Privacy Policy.</a> Powered by <a href="https://ezplatform.com/">eZ Platform</a></li>
```
Comment (nicolas-bastien) [inline app/Resources/views/pagelayout_footer.html.twig]: @DavidLiedle I already fix that in this PR https://github.com/ezsystems/ezplatform-demo/pull/47

**Comment by nicolas-bastien** [comment]
@clash82 I guess this shouldn't be merge as fotter modification will be done here https://github.com/ezsystems/ezplatform-demo/pull/47

**Comment by clash82** [comment]
You're right @nicolas-bastien, all of this was already done by you.

**Pull request closed without merging**